test/e2e/knit_podres_test.go

```diff
@@ -0,0 +1,29 @@
+package e2eknit
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+)
+
+var _ = g.Describe("knit podresources tests", func() {
```

Comment:
maybe we can avoid the explicit calls here  g.Describe -> Describe

Reply:
That was intentional to avoid namespace pollution. Do we prefer to import identifiers in the current namespace?

Reply:
atm I'm in favour of keeping the namespace qualifications, but we will re-evaluate (and fix all tests in one go) if this becomes too noisy indeed. Thanks for raising this point!